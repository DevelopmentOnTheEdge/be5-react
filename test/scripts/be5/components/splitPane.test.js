import React    from 'react';
import renderer from 'react-test-renderer';
import SplitPane     from  '../../../../src/scripts/be5/components/splitPane';
import be5 from '../../../../src/scripts/be5/be5';

it('snapshot menu', () => {

  const component = renderer.create(
    <SplitPane split="vertical" defaultSize={280} >
      <div/>
      <div/>
    </SplitPane>
  );

  expect(component.toJSON()).toMatchSnapshot();
});