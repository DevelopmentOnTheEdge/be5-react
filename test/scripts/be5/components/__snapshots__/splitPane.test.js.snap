// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot menu 1`] = `
<div
  className="SplitPane vertical"
  style={
    Object {
      "display": "flex",
      "flex": 1,
      "flexDirection": "row",
      "height": "100%",
      "left": 0,
      "outline": "none",
      "overflow": "hidden",
      "position": "absolute",
      "right": 0,
    }
  }
>
  <div
    className="Pane vertical pane1"
    style={
      Object {
        "flex": "none",
        "outline": "none",
        "overflow": "auto",
        "position": "relative",
        "width": 280,
      }
    }
  >
    <div />
  </div>
  <span
    className="Resizer vertical"
    onMouseDown={[Function]}
  />
  <div
    className="Pane vertical pane2"
    style={
      Object {
        "flex": 1,
        "outline": "none",
        "overflow": "auto",
        "position": "relative",
      }
    }
  >
    <div />
  </div>
</div>
`;
